pay-api/gunicorn_config.py

```diff
@@ -18,10 +18,10 @@
 import os
 
 # https://docs.gunicorn.org/en/stable/settings.html#workers
-workers = int(os.environ.get('GUNICORN_PROCESSES', '3'))  # gunicorn default - 1
+workers = int(os.environ.get('GUNICORN_PROCESSES', '1'))  # gunicorn default - 1
```

Comment:
Should fix the issue where 3 migrators try to upgrade the database at the same time